designs/unice/src/front.js

```diff
@@ -163,14 +182,57 @@ export default function (part) {
   points.frontGussetLeftCp1 = points.frontGussetLeft
     .shift(270, Math.max(Math.max(points.frontGussetLeft.dy(points.frontWaistMid) * options.taperToGusset / 2,points.frontGussetLeft.dy(points.frontLegOpeningLeft) * 2),points.frontGussetLeft.dy(points.frontWaistBandMid)))
 
+  // straighten the part (if needed)
+  let curveAsAngle
+  if (options.frontCurve > 0) {
+    // create copies of the original points
+    points.frontWaistBandLeftPreCurve = points.frontWaistBandLeft
+    points.frontLegOpeningLeftPreCurve = points.frontLegOpeningLeft
+    points.frontLegOpeningLeftCp1PreCurve = points.frontLegOpeningLeftCp1
```

Comment:
Ah, thanks, I did not know that!

Is it okay to keep these lines of code (after adding the .clone()) even though the points are not used anywhere? I think I might need them later, but I'm not sure whether that's enough of a reason to keep them.

Reply:
Yes, you can keep the lines of code if you believe having the points will be useful in the future, even if the design doesn't use them now. Doing this will also help other pattern designers understand what your design is doing when they read your code.